spi.cpp

```diff
@@ -553,10 +568,11 @@ int InitSPI()
   // Set the SPI 0 pin explicitly to output, and enable chip select on the line by setting it to low.
   // fbcp-ili9341 assumes exclusive access to the SPI0 bus, and exclusive presence of only one device on the bus,
   // which is (permanently) activated here.
-  SET_GPIO_MODE(GPIO_SPI0_CE0, 0x01);
-  CLEAR_GPIO(GPIO_SPI0_CE0);
+  SET_GPIO_MODE(GPIO_SPI0_CE1, 0x01);

```

Comment:
Probably didn't mean to permanently change this to CS1 outside of the `#ifdef DISPLAY_USES_CS1` below?